EnglishFilter/sections/specific.txt

```diff
@@ -1151,6 +1151,12 @@ collive.com##.premium-post-ad
 m.xcum.com##.mob_under_player
 roadtovr.com##footer > div.td-post-content
 nubng.com,techhelpbd.com###stream-item-widget-5
+compstudio.in##.stream-item-top-wrapper
+compstudio.in##.stream-item-above-post
+compstudio.in##.stream-item-above-post-content
+compstudio.in##.stream-item-widget
+compstudio.in##.stream-item-inline-post
+compstudio.in##.stream-item-below-post-comments
```

Comment:
I'd say you can use `compstudio.in##div[class^="stream-item"]`, but please check this (it's ok on my end).

Reply:
I thought about what you said. I thought this version was more specific. OK, I'll change it to attribute selector